docs/sdk/examples/pastefromlibreoffice.html

```diff
@@ -0,0 +1,295 @@
+				// The full preset from CDN which we used as a base provides more features than we need.
+				// Also by default it comes with a 3-line toolbar. Here we put all buttons in two rows.
+				toolbar: [
+					{ name: 'clipboard', items: [ 'Paste', '-', 'Undo', 'Redo' ] },
+					{ name: 'basicstyles', items: [ 'Bold', 'Italic', 'Underline', 'Strike', 'RemoveFormat', 'Subscript', 'Superscript' ] },
+					{ name: 'links', items: [ 'Link', 'Unlink' ] },
+					{ name: 'paragraph', items: [ 'NumberedList', 'BulletedList', '-', 'Outdent', 'Indent', '-', 'Blockquote' ] },
+					{ name: 'insert', items: [ 'Image', 'Table' ] },
+					{ name: 'editing', items: [ 'Scayt' ] },
+					'/',
+
+					{ name: 'styles', items: [ 'Format', 'Font', 'FontSize' ] },
+					{ name: 'colors', items: [ 'TextColor', 'BGColor', 'CopyFormatting' ] },
+					{ name: 'align', items: [ 'JustifyLeft', 'JustifyCenter', 'JustifyRight', 'JustifyBlock' ] },
+					{ name: 'document', items: [ 'Print', 'PageBreak', 'Source' ] }
+				],
+
+				// Since we define all configuration options here, let's instruct CKEditor to not load config.js which it does by default.
+				// One HTTP request less will result in a faster startup time.
+				// For more information check https://ckeditor.com/docs/ckeditor4/latest/api/CKEDITOR_config.html#cfg-customConfig
+				customConfig: '',
+
+				// Upload images to a CKFinder connector (note that the response type is set to JSON).
+				uploadUrl: '/apps/ckfinder/{%CKFINDER_VERSION%}/core/connector/php/connector.php?command=QuickUpload&type=Files&responseType=json',
+
+				// Configure your file manager integration. This example uses CKFinder 3 for PHP.
+				filebrowserBrowseUrl: '/apps/ckfinder/{%CKFINDER_VERSION%}/ckfinder.html',
+				filebrowserImageBrowseUrl: '/apps/ckfinder/{%CKFINDER_VERSION%}/ckfinder.html?type=Images',
+				filebrowserUploadUrl: '/apps/ckfinder/{%CKFINDER_VERSION%}/core/connector/php/connector.php?command=QuickUpload&type=Files',
+				filebrowserImageUploadUrl: '/apps/ckfinder/{%CKFINDER_VERSION%}/core/connector/php/connector.php?command=QuickUpload&type=Images',
+
+				// Sometimes applications that convert HTML to PDF prefer setting image width through attributes instead of CSS styles.
+				// For more information check:
+				//  - About Advanced Content Filter: https://ckeditor.com/docs/ckeditor4/latest/guide/dev_advanced_content_filter
+				//  - About Disallowed Content: https://ckeditor.com/docs/ckeditor4/latest/guide/dev_disallowed_content
+				//  - About Allowed Content: https://ckeditor.com/docs/ckeditor4/latest/guide/dev_allowed_content_rules
+				disallowedContent: 'img{width,height,float}',
+				extraAllowedContent: 'img[width,height,align];span{background}',
+
+				// Enabling extra plugins, available in the full-all preset: https://ckeditor.com/cke4/presets
+				extraPlugins: 'colorbutton,font,justify,print,tableresize,uploadimage,uploadfile,pastefromword,liststyle,pagebreak',
```

Comment:
I'm wondering if we need `pastefromword` here, but probably shouldn't hurt, especially that someone could just open incorrect sample when checking out how PFW works. Maybe it makes sense to add `pastefromgdocs` also?

On the other hand, shouldn't these plugins be added automatically by preset?